internal/http/client.go

```diff
@@ -84,11 +83,15 @@ func NewClient(config config.Config) NewRelicClient {
 
 		level, err := log.ParseLevel(config.LogLevel)
 		if err != nil {
-			panic(err)
+			log.Warn(fmt.Sprintf("could not parse log level '%s', logging will proceed at info level", config.LogLevel))
+			level = log.InfoLevel
```

Comment:
This should output `defaultLogLevel` and parse that level again since we have a constant, easy to miss this if we change the default level later.